frontend/src/pages/notebookController/screens/server/SpawnerPage.tsx

```diff
@@ -445,24 +455,35 @@ const SpawnerPage: React.FC = React.memo(() => {
               {` Add more variables`}
             </Button>
           </FormSection>
-          <ActionGroup>
-            <Button
-              variant="primary"
-              onClick={() => {
-                handleNotebookAction().catch((e) => {
-                  setCreateInProgress(false);
-                  setStartShown(false);
-                  console.error(e);
-                });
-              }}
-              isDisabled={createInProgress}
-            >
-              Start server
-            </Button>
-            <Button variant="secondary" onClick={() => history.push('/')}>
-              Cancel
-            </Button>
-          </ActionGroup>
+          <div>
```

Comment:
May I ask why we need this `div` here?

Reply:
Funky way the forms are built with spacing between each item, makes it have ungodly white spaces around everything if they don't share the same singular DOM node in the form.

I logged https://github.com/opendatahub-io/odh-dashboard/issues/396 to take this further, hopefully cleaning up this random `div` and make this an official form footer.